IFS/tco1279-ng5-production.yaml

```diff
@@ -0,0 +1,17 @@
+plugins:
+  source:
+    - module: intake_xarray
+    - module: gribscan
+
```

Comment:
```suggestion
```

With the release of Intake2 the `plugins` section is no longer supported. Both `intake_xarray` and `gribscan` add an entry points so that intake is still able to find them, so this part can simply be dropped (see #25 )